charts/kamu-api-server/values.yaml

```diff
@@ -49,6 +49,7 @@ imageMigrations:
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+enableDatabase: false
```

Comment:
**Q:** why do we need this switch? we want to do a [future] deployment on `demo` without a DB?

Reply:
Yes. exactly. Currently only europort env will be with database